src/ContainerRemoveErrorModal.jsx

```diff
@@ -17,7 +17,7 @@ const ContainerRemoveErrorModal = (props) => {
                 </ModalBody>
                 <ModalFooter>
                     <Button color="secondary" onClick={props.handleCancelRemoveError}>{_("Cancel")}</Button>
-                    <Button color="danger" onClick={props.handleForceRemoveContainer}>{_("Force Delete")}</Button>{' '}
+                    <Button id="confirmation-dialog-confirm" className={props.setWaitCursor} color="danger" onClick={props.handleForceRemoveContainer}>{_("Force Delete")}</Button>{' '}
```

Comment:
This ID is actively dangerous. You set exactly the same one in another component further down, and thus would have two different elements with the same ID. You also don't use it anywhere, so please remove these two.